build/builders.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Using Docker Build
+description: About building with existing builders
+keywords: build, buildx, buildkit
+---
+
+
+## Docker Build Components
+
+Currently Engine holds two build components:
+* Legacy Builder - Old builder implementation. It's executed through the `docker build` command.
+* Buildkit - Active by default in new and latest installations of Docker. It's executed through the `docker buildx build` command.
```

Comment:
This still depends on the configuration of the env var. So this text needs to be revisited even or current scenario (if well understood).